lib/controllers/rsvps.dart

```diff
@@ -119,9 +123,13 @@ class InstanceRsvpsController
     state = AsyncValue.data(await rsvpsController.hydrate(data));
   }
 
-  Future<InstanceMember?> save(InstanceMemberStatus? status) async {
+  Future<InstanceMember?> save(
+    InstanceMemberStatus? status,
+    Instance instance,
+  ) async {
```

Comment:
@IguJl15 see other note on editor auto-reformatting conflicts. Let's get noise like this eliminated from our commits. I'm using pretty stock settings, do you have prettifier running too?